### DISCUSSION TITLE
Can I safely update to 1.37 if I have Dongle-E with Multi-Pan (RCP)?

### DISCUSSION THREAD

**Discussion by SDeGonge** [Koenkk/zigbee2mqtt#22555] [Devices]
I'm a little nervous about updating from Z2M 1.36.1-1 to 1.37.0-1...

I currently have the following:
- zigbee2mqtt 1.36.1-1
- Sonoff Dongle-E
- **_Multi-PAN (RCP) 4.3.1_**
- Configuration.yaml --> adapter: ezsp

I'm under the impression that updating to 1.37 may not work if I am using Multi-PAN firmware. Is this correct? Can I just update to the latest Z2M version 1.37 or is there more that must be done? Thanks

**Comment by Grogger** [comment]
It works fine on my end. I didn't switch my adapter to ember though, I still have it as ezsp.
